### PR TITLE
Update haproxy.rst

### PR DESCRIPTION
**Pull request by superwinni2**
A lot of people don't understand how many Frontends they need to create... Hopefully this is more understandable.
Also: Updated to Conditions / Rules / Public Services Names

### CODE REVIEW

File: source/manual/how-tos/haproxy.rst [hunk 1]
```diff
@@ -103,22 +106,27 @@ In the open modal dialog, the following form will show up:
...
 
---------------------------------
-Fourth Step: Configure an Action
---------------------------------
+---------------------------------------
+Fourth Step: Configure an Rule
+---------------------------------------
+
+As promised in the previous step, the Conditions will be used. 
+A Rule can use multiple conditions to dicide whcih Rule is going to be used.
```
Comment (fraenki) [inline source/manual/how-tos/haproxy.rst]: two typos, "decide which"

File: source/manual/how-tos/haproxy.rst [hunk 2]
```diff
@@ -136,12 +144,16 @@ Fifth Step Configure a frontend
...
 
 Now its nearly done. The only thing that needs to be configured for HAProxy
-is a Frontend.
-A Frontend is a a group of bound ports which are used for incoming connections.
-From this Frontend we need to know which backend the request will routed to.
+is a Public Service.
+A Public Service is a a group of bound ports which are used for incoming connections.
+From this Public Service we need to know which backend the request will routed to.
 For this, the previously configured action is needed.
+If you got multiple domains on one IP, you differate them with rules!
```
Comment (fraenki) [inline source/manual/how-tos/haproxy.rst]: typo, "differentiate"

File: source/manual/how-tos/haproxy.rst [hunk 3]
```diff
@@ -177,8 +189,9 @@ the client.
 Actions (ACLs)
 ==============
 
-Here you have to configure the previously configured actions, so HAProxy
-knows where the requests should be sent to.
+Here you have to activate the previously configured actions, so HAProxy
+is going to operate based due the rules/actions.
```
Comment (fraenki) [inline source/manual/how-tos/haproxy.rst]: "rules/actions" should be "rules/conditions" to align with the new wording :)

File: source/manual/how-tos/haproxy.rst [hunk 4]
```diff
@@ -177,8 +189,9 @@ the client.
...
+knows where the requests should be sent to.
```
Comment (fraenki) [inline source/manual/how-tos/haproxy.rst]: looks like a leftover from the old version... should be removed?

**Comment by fraenki** [comment]
@superwinni2 Thanks for your contribution! A few minor changes and this would be ready for merge :)

**Comment by fichtner** [comment]
@superwinni2 please address review notes by @fraenki so we can merge it :)

File: source/manual/how-tos/haproxy.rst (hunk 2 above)
Comment (superwinni2) [reply source/manual/how-tos/haproxy.rst]: a bad english speaking german guy writing in english :D 
Thanks!

File: source/manual/how-tos/haproxy.rst (hunk 3 above)
Comment (superwinni2) [reply source/manual/how-tos/haproxy.rst]: Jep you're right ;) :+1:

File: source/manual/how-tos/haproxy.rst (hunk 4 above)
Comment (superwinni2) [reply source/manual/how-tos/haproxy.rst]: Would also say that it's a leftover from the past :)
Thanks

**Comment by fichtner** [comment]
Merged, thank you! ❤️

**Pull request merged**